ParkingZoneApp/Repository/Interfaces/IParkingSlotRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using ParkingZoneApp.Models.Entities;
+
+namespace ParkingZoneApp.Repository.Interfaces
+{
+    public interface IParkingSlotRepository : IRepository<ParkingSlot>
+    {
+        ICollection<ParkingSlot> GetAllParkingSlots();
```

Comment:
in service we added GetAll() instead of GetAllParkingSlots() thats why we dont need it